### PR TITLE
More typos/translation fixes

### PR DESCRIPTION
**Pull request by Crevox**
The skill is the Daino buff/status description.

Changed "increases" to "increased" because it matches what we do everywhere else.

### CODE REVIEW

File: EnglishTranslation-master/ITEM.tsv [hunk 1]
```diff
@@ -5609,7 +5609,7 @@ ITEM_20150717_005608	Don't think it is poor just because of its form. It has a s
 ITEM_20150717_005609	The Battle ways using this unique form of the shield has been upgraded by lots of Masters.
 ITEM_20150717_005610	In the fast, There were a lot of shield crafted by Fedimian. Since Medzio Diena, the lot of type shield have been crafted due to the dangerous world.
 ITEM_20150717_005611	Unlike the some of misunderstanding, This type is not origin from the kingdom outside. Since Medzio Diena, It made an effect, surely the shield is origin from the some place of the kingdom.
-ITEM_20150717_005612	This shield is specilized in riding battle, It can make a good performance at the Hamasi battle. After Taniel period which made a stablization Kingdom, It has been crafted constanly.
+ITEM_20150717_005612	This shield is specialized for mounted battles. It performed well at the Hamasi battle. After the Taniel period which stabilized the kingdom, it has been produced constantly.
```
Comment (Sunaries) [inline EnglishTranslation-master/ITEM.tsv]: "battle of Hamasi"

**Comment by Sunaries** [comment]
Update ETC_20150428_011426 as well to reflect the correct name (Cavalry).

**Comment by Crevox** [comment]
Should be good now.

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Pull request merged**